test/e2e/snaps/test-snap-multi-install.spec.js

```diff
@@ -0,0 +1,111 @@
+const {
+  defaultGanacheOptions,
+  withFixtures,
+  unlockWallet,
+  WINDOW_TITLES,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+const { TEST_SNAPS_WEBSITE_URL } = require('./enums');
+
+describe('Test Snap Multi Install', function () {
+  it('test multi install snaps', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions: defaultGanacheOptions,
+        failOnConsoleError: false,
+        title: this.test.fullTitle(),
+      },
+      async ({ driver }) => {
+        await unlockWallet(driver);
+
+        // navigate to test snaps page and multi-install snaps
+        await driver.openNewPage(TEST_SNAPS_WEBSITE_URL);
+        await driver.delay(1000);
```

Comment:
Can we wait for selector here instead of using fixed delay?

Reply:
This fixed delay is used in all of the tests - much like the other comment, the tests flake as a result of timing delays before looking for a selector, and it will timeout even if the selector is visible after the waitforselector is called. It's one of the two situations the delays can't be helped that I can think of at the moment.

Reply:
Ok, understood. If it resolves flaky test issues, let it be.